src/zzz_od/gui/view/setting/setting_plugin_interface.py

```
@@ -0,0 +1,674 @@
+                duration=5000,
+                parent=self
+            )
+
+    def _on_open_dir_clicked(self) -> None:
+        """打开插件目录"""
+
+        plugins_dir = self.plugin_import_service.plugins_dir
+        plugins_dir.mkdir(parents=True, exist_ok=True)
+
+        if sys.platform == 'win32':
+            os.startfile(plugins_dir)
```

Ruff (0.16.1)

[error] Starting a process without a shell

(S606)